src/utils/sprint-utils.ts

```diff
@@ -117,9 +117,25 @@ export const getSeveraUserId = (user: User): string => {
  * @returns date in format of "dd MMM yyyy"
  */
 export const formatDateSevera = (date: string) => {
-  return new Date(date).toLocaleDateString("en-US", {
+  return new Date(date).toLocaleDateString("fi-FI", {
     day: "2-digit",
-    month: "short",
+    month: "numeric",
     year: "numeric",
   });
+};
+
+export const mapPhasesToRows = (phase: Phase, workHours: WorkHours[]) => {
+  const actualWorkHours = getWorkHour(
+    phase.severaPhaseId || "Severa Phase Id not found",
```

Comment:
Not sure why you are passing this "Severa Phase Id not found" string as the id in some cases. Can you explain?